src/orca/services/nextflowtower/models.py

```diff
@@ -223,7 +224,7 @@ def to_json(self) -> dict[str, Any]:
             "configProfiles": dedup(self.profiles),
             "configText": self.nextflow_config,
             "dateCreated": None,
-            "entryName": None,
```

Comment:
I see that this used to be None, can you clarify the hardcoding bit when setting this value to be "".

Reply:
If I set `entry_name: Optional[str] = None` instead of `""`, if you aren't using the `entry_name` variable when you create your `LaunchInfo` object it triggers [this](https://github.com/Sage-Bionetworks-Workflows/py-orca/blob/9a8483a5a30ac2c5174ea228b26eeb2dcf23607f/src/orca/services/nextflowtower/models.py#L96) error when you try to launch the workflow, unless you include a line similar to 
```
launch_info.fill_in("compute_env_id", compute_env.id)
```
for `entry_name` in `launch_workflow`. But since that line would look like
```
launch_info.fill_in("entry_name", "")
```
I figured it was better to make the default `""`.

Reply:
Ahh I see. Makes sense! Thanks for the explanation.